webdriver/tests/bidi/__init__.py

```diff
@@ -11,7 +11,7 @@ def recursive_compare(expected: Any, actual: Any) -> None:
         expected(actual)
         return
 
-    assert type(expected) == type(actual)
+    assert type(expected) is type(actual)
```

Comment:
what is the difference between the two methods?

Reply:
There was a linter warning (from pylance I think) suggesting to use `is` or `isinstance` for types.

For cases without inheritance I believe it's a no-op. I just did this to silence the warning.

Some context here: https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance

Reply:
Given the above: Any other concerns?